packages/rancher-windows-gmsa/charts/templates/_helpers.tpl

```diff
@@ -0,0 +1,28 @@
+# Rancher
+
+{{- define "system_default_registry" -}}
+{{- if .Values.global.cattle.systemDefaultRegistry -}}
+{{- printf "%s/" .Values.global.cattle.systemDefaultRegistry -}}
+{{- end -}}
+{{- end -}}
+
+{{/* Create chart name and version as used by the chart label. */}}
+{{- define "gmsa.chartref" -}}
+{{- replace "+" "_" .Chart.Version | printf "chart: %s-%s" .Chart.Name -}}
+{{- end }}
+
+{{/* Determine ApiVersion for Cert-manager */}}
+{{- define "cert-manager.apiversion" -}}
+  {{- $certmanagerVer :=  split "." .Values.certificates.certManager.version -}}
+  {{- if or (.Capabilities.APIVersions.Has "cert-manager.io/v1") (and (gt (len $certmanagerVer._0) 0) (eq (int $certmanagerVer._0) 1) (ge (int $certmanagerVer._1) 0)) }}
+{{- printf "apiVersion: cert-manager.io/v1" -}}

```

Comment:
you don't need to printf, you could just put `apiVersion: cert-manager.i/v1` directly with no brackets